activesupport/lib/active_support/message_encryptor.rb

```diff
@@ -87,14 +88,18 @@ def initialize(secret, *signature_key_or_options)
 
     # Encrypt and sign a message. We need to sign the message in order to avoid
     # padding attacks. Reference: http://www.limited-entropy.com/padding-oracle-attacks.
-    def encrypt_and_sign(value)
+    def encrypt_and_sign(value, expires_at: nil, expires_in: nil, purpose: nil)
+      if expires_at || expires_in || purpose
```

Comment:
Let's move this `if` into `wrap` and make it return the original message in case no metadata options have been passed. It makes sense for the metadata object to handle whether or not to wrap — and we're going to need this `if` for the verifier as well.